evennia/utils/test_resources.py

```diff
@@ -66,3 +66,42 @@ def tearDown(self):
         self.account.delete()
         self.account2.delete()
         super(EvenniaTest, self).tearDown()
+
+
+import unittest
+import inspect
+
+class TestCreateScript(EvenniaTest):
+
+    def setUp(self):
+        super(EvenniaTest, self).setUp()
+
+    def tearDown(self):
+        super(EvenniaTest, self).tearDown()
+
+    def test_create_script(self):
+        class TestScriptA(DefaultScript):
+            def at_script_creation(self):
+                self.key = 'test_script'
+                self.interval = 10
+                self.persistent = False
+
+        script = create.create_script(TestScriptA, key='test_script')
```

Comment:
This will cause problems down the line due to the async execution of scripts (basically the callback will fire after the test method already concluded). You should probably make sure to run script.stop() at the end of all methods using scripts with an interval.

Reply:
Hm. I don't understand what you mean.

What is async in this code? the database save is async?
What `callback` are you referring to?
Where should I call `script.stop()`? After `create.create_script()` returns?

Reply:
The entire repeating Script is async. The callback is the `at_repeat` method of the Script. Which, unless properly handled, may fire 10 seconds after this test-script was created, at which point the test suite has long since moved on to something else. Or worse, may have concluded and shut down the event loop already. So make sure to stop your script, for example in the tearDown method or yes, at the end of your test body.

Reply:
Will do that!

Of course, that won't work for the *failing* test-case `repeats = 1` as the `create_script` function never gets to return.